### PR TITLE
Generate code for ECS 8.11

### PR DESCRIPTION
**Pull request by Mpdreamz**

### CODE REVIEW

**Comment by Mpdreamz** [comment]
Running into this: https://github.com/elastic/ecs/issues/2326

File: tools/Elastic.CommonSchema.Generator/Projection/IndexTemplate.cs [hunk 1]
```diff
@@ -32,9 +32,11 @@ public IndexTemplate(string name, string template, string schemaVersion)
 					.Replace("\"priority\": 1,", $"\"priority\": {Priority},\r\n  \"data_stream\": {{}},")
 					.Replace("Sample composable template that includes all ECS fields",
 						$"Template installed by ECS.NET {schemaVersion} (https://github.com/elastic/ecs-dotnet)")
+					.Replace("\"limit\": 2000", "\"limit\": 2500")
 					.Replace("\"", "\"\"")
 					.Replace("_vulnerability\"\"", "_vulnerability\"\"\" + userComponents + @\"")
-					.Replace("try-ecs-*", "\" + indexPattern + @\"");
+					.Replace("try-ecs-*", "\" + indexPattern + @\"")
+					;
```
Comment (stevejgordon) [inline tools/Elastic.CommonSchema.Generator/Projection/IndexTemplate.cs:39]: nit: did we mean to push the semicolon to a new line?

**Review by stevejgordon — DISMISSED** [review]
LGTM with one minor nit

File: tools/Elastic.CommonSchema.Generator/Projection/IndexTemplate.cs (hunk 1 above)
Comment (Mpdreamz) [reply tools/Elastic.CommonSchema.Generator/Projection/IndexTemplate.cs]: Nope good catch! :)

**Review by stevejgordon — APPROVED** [review]
lgtm

**Pull request merged**